### DISCUSSION TITLE
Welcome to BoyosBot Discussions!

### DISCUSSION THREAD

**Discussion by NoahFlowa** [NoahFlowa/BoyosBot#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wymillerlinux** [comment]
Howdy! I'm Wyatt, and I'm the infrastructure guy and contributor to the BoyosBot, which means I help run the bot and fix @NoahFlowa's bugs when I find them. When I'm not developing the bot, you can find me either developing software in exchange for food, on the beach, floating down the river in a kayak, or camping!
